apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -152,6 +152,11 @@ function load_posts_list_block() {
  * Load Starter_Page_Templates.
  */
 function load_starter_page_templates() {
+	// We don't want the user to choose a template when copying a post.
+	// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+	if ( isset( $_GET['jetpack-copy'] ) ) {
+		return;
+	}
```

Comment:
Minor: It would be nice to have an empty line after this block to visually break them up